packages/vue-client/src/playing_table_map.ts

```diff
@@ -0,0 +1,5 @@
+import { Component } from "vue";
+
+export const playing_table_map: {
```

Comment:
Possibly add comment here describing the purpose of the components here, and the relationship to board_map